src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -110,7 +110,7 @@ function ButtonWithDropdownMenu<IValueType>({
                         isDisabled={isDisabled || !!selectedItem?.disabled}
                         isLoading={isLoading}
                         shouldRemoveRightBorderRadius
-                        style={[styles.flex1, styles.pr0]}
+                        style={isSplitButton ? [styles.flex1, styles.pr0] : {}}
```

Comment:
I decided to use this solution because if we always apply flexRow, the button will be broken in the import page.
<img width="437" alt="Screenshot 2024-09-04 at 19 52 02" src="https://github.com/user-attachments/assets/225eb186-26c0-4635-80b3-a3169cc1d850">

Reply:
Additional screenshot just to make sure ButtonWithDropdownMenu with `isSplitButton` is still rendered correctly.
<img width="315" alt="image" src="https://github.com/user-attachments/assets/09024d6e-9b25-44e6-a8c2-3940ef5d0bf0">
<img width="411" alt="image" src="https://github.com/user-attachments/assets/6556a395-2b35-455d-a62e-f5fc6d061f03">